wiki/Difficulties/en.md

```diff
@@ -45,7 +42,7 @@ osu!taiko difficulty spreads
 
 From the [Ranking Criteria](/wiki/Ranking_Criteria):
 
--   **The difficulties in the mapset must be in a consecutive order.** Easy or Normal can be skipped if the gap in the star rating spread allows it. If your mapset has two difficulties, one of them cannot be an Insane or Expert. The lowest difficulty must be below 2.0 stars.
+> -   **The difficulties in the mapset must be in a consecutive order.** Easy or Normal can be skipped if the gap in the star rating spread allows it. If your mapset has two difficulties, one of them cannot be an Insane or Expert. The lowest difficulty must be below 2.0 stars.
 -   **A mapset cannot have just one osu!taiko difficulty.** If you decide to include osu!taiko in your map, make sure you have at least 2 osu!taiko mode difficulties and that one of them is of an easier difficulty level if it doesn't have an osu! standard Easy level difficulty.
 
 A common set contains the following four difficulties: Kantan, Futsuu, Muzukashii and Oni which difficulty spread between them **should be approximately kept the same**. To achieve this, it is highly recommended to set a **main snapping** for each difficulty which **follows the pace of the song**. If the spread still happens to be uneven, you may move to another type of set which contains an additional Inner Oni and sometimes even an Ura Oni. These difficulties can be used as filler in a mapset to ensure a consistent spread in case four difficulties are not enough. Also in this way, an **Oni can be mapped easier for people who are still not familiar with harder difficulties, while Inner/Ura serves as harder Oni**.
```

Comment:
- `Also in this way, an **Oni` -> `Also in this way, **Oni`
- `while Inner/Ura serves as harder Oni**.` -> `while Ura serves as harder Oni**.`